decentraland/unity-renderer
# feat: Add get friends listener

## What does this PR change?



Closes decentraland/dservices#68

## How to test the changes?



1. Launch the explorer
2. Enter the friends tab, it should show all current friends 

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
